### PR TITLE
feat(image): facade api

### PR DESCRIPTION
**Pull request by DR-Univer**
## Pull Request Checklist

- [x] Related tickets or issues have been linked in the PR description (or missing issue).
- [x] [Naming convention](https://github.com/dream-num/univer/blob/dev/docs/NAMING_CONVENTION.md) is followed (**do please** check it especially when you created new plugins, commands and resources).
- [x] Unit tests have been added for the changes (if applicable).
- [x] Breaking changes have been documented (or no breaking changes introduced in this PR).

### CODE REVIEW

File: packages/core/src/facade/f-blob.ts [hunk 1]
```diff
@@ -0,0 +1,197 @@
...
+                resolve(text);
+            }).catch((error) => {
+                reject(new Error(`Failed to read Blob as ArrayBuffer: ${error.message}`));
+            });
+        });
+    }
+
+    /**
+     * Gets the data stored in this blob.
+     * @returns a byte array
```
Comment (VicKun4937) [inline packages/core/src/facade/f-blob.ts:114]: @return {string}  -The blob byte array.

File: packages/core/src/facade/f-blob.ts [hunk 2]
```diff
@@ -0,0 +1,197 @@
...
+     * ```ts
+     * const blob = UniverApi.newBlob();
+     * blob.setDataFromString('Hello, World!');
+     * ```
+     */
+    setDataFromString(data: string): void;
+    /**
+     * Sets the data stored in this blob.
+     * @param data a string
+     * @param contentType the content type refer to https://developer.mozilla.org/en-US/docs/Web/HTTP/MIME_types/Common_types
```
Comment (VicKun4937) [inline packages/core/src/facade/f-blob.ts:156]: the optional param should comment like following:
@param {string} [contentType] - The contentType, which refer to  https://developer.mozilla.org/en-US/docs/Web/HTTP/MIME_types/Common_types

File: packages/sheets/src/basics/cell-position.ts [hunk 3]
```diff
@@ -0,0 +1,27 @@
...
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+export interface ICellOverGridPosition {
+    column: number; // column number
+    columnOffset: number; // column offset, unit is EMUs
```
Comment (VicKun4937) [inline packages/sheets/src/basics/cell-position.ts:19]: The offset is not EMU unit, please pay attention.

File: packages/sheets-drawing-ui/src/facade/f-worksheet.ts [hunk 4]
```diff
@@ -60,6 +158,138 @@ export class FWorksheetLegacy extends FWorksheet implements IFWorksheetLegacy {
...
+        const imageBuilder = this.newOverGridImage();
+        if (typeof url === 'string') {
+            imageBuilder.setSource(url);
+        } else {
+            const blobSource = url.getBlob();
+            const base64 = await blobSource.getDataAsString();
+            imageBuilder.setSource(base64, ImageSourceType.BASE64);
+        }
+
+        if (column != null) {
```
Comment (VicKun4937) [inline packages/sheets-drawing-ui/src/facade/f-worksheet.ts:172]: why not  !==

File: packages/sheets/src/basics/cell-position.ts [hunk 5]
```diff
@@ -0,0 +1,27 @@
...
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+export interface ICellOverGridPosition {
+    column: number; // column number
```
Comment (wzhudev) [inline packages/sheets/src/basics/cell-position.ts:18]: Use JSDoc format as well?

File: packages/sheets-drawing-ui/src/facade/f-worksheet.ts [hunk 6]
```diff
@@ -33,6 +35,102 @@ export interface IFWorksheetLegacy {
...
+
+     *  setTimeout(async ()=>{
+     *   const builder = image.toBuilder();
+     *   const param = await builder.setHeight(50).setWidth(100).build();
+     *   activeSheet.updateImages([param]);
+     *  }, 4000);
+     */
+    insertImages(sheetImages: ISheetImage[]): void;
+
+    getImages(): FOverGridImage[];
```
Comment (wzhudev) [inline packages/sheets-drawing-ui/src/facade/f-worksheet.ts:86]: JSDoc required.

**Review by wzhudev — APPROVED** [review]

**Review by VicKun4937 — APPROVED** [review]

**Review by jikkai — APPROVED** [review]

File: packages/sheets/src/basics/cell-position.ts (hunk 5 above)
Comment (DR-Univer) [reply packages/sheets/src/basics/cell-position.ts:18]: 先合入，后续我补一个

File: packages/sheets-drawing-ui/src/facade/f-worksheet.ts (hunk 6 above)
Comment (DR-Univer) [reply packages/sheets-drawing-ui/src/facade/f-worksheet.ts:86]: 先合入，后续我补一个

Comment (DR-Univer) [reply packages/sheets-drawing-ui/src/facade/f-worksheet.ts:86]: 先合入，后续我补一个

**Pull request merged**